packages/side-nav/src/SideNav.tsx

```diff
@@ -96,12 +87,6 @@ export function SideNav({
 					fontFamily="body"
 					fontSize="sm"
 					lineHeight="default"
-					css={mq({
```

Comment:
We had to move this CSS declaration logic up a level to side nav container, otherwise the collapse button doesn't have access to the variables.